cypress.config.ts

```diff
@@ -11,21 +11,6 @@ export default defineConfig({
   },
   viewportHeight: 900,
   viewportWidth: 1440,
-  env: readEnvironmentVariables(),
```

Comment:
This is useful you should consider leaving it in

Reply:
I can leave it but it will be a function returning empty content, is that fine?

Reply:
I suppose we should remove useless logic!

Reply:
Well these are not needed for now since it's mocked. We can always find the logic back within the Git history if needed, so will remove it for now.